accelerators-home/modules/ROOT/pages/getting-started.adoc

```diff
@@ -31,32 +31,34 @@ All accelerator assets are available from the main Anypoint Exchange site and ar
 
 The following instructions assume you have already configured your Maven `settings.xml` file according to the xref:build-environment.adoc[build environment configuration] guide.
 
-==== Import Directly from Anypoint Exchange
+==== Import Directly from Anypoint Studio or Import from Anypoint Code Builder
 
```

Comment:
Also, it would be better to have two separate sections, one for Import into Studio and one for Import into ACB and then Studio would have two options, one to import directly from Exchange and one to import manually and for ACB, we wouldn't need two sections as there is only one way to do it.